cypress/e2e/default/rewrites-redirects.cy.ts

```diff
@@ -1,5 +1,6 @@
 describe('Rewrites and Redirects', () => {
-  it('rewrites: points /old to /', () => {
+  // TODO: detect if rewrite is app or page route for __NEXT_PRIVATE_PREBUNDLED_REACT logic
```

Comment:
it's for a future one - we can't currently support this use case because of the long-standing redirect stacking issue https://github.com/netlify/pillar-runtime/issues/125